OmniBLE/OmnipodCommon/BeepPreference.swift

```diff
@@ -25,13 +26,18 @@ public enum BeepPreference: Int, CaseIterable {
     }
 
     var description: String {
+        // This picks up the DefaultValue from LoopKit, not the CFBundleDisplayName
+        //@Environment(\.appName) var appName
+        // ToDo - insert the appName properly
+        let appName: String = "the app"
```

Comment:
Why not use the environment with the real app name? If you have questions about how to set it, I can help. If you want to use "the app", it would be better just to hard code it.

Reply:
Please help. I prefer appName.